launch/personal_food_computer_v2.launch

```diff
@@ -1,5 +1,6 @@
 <launch>
   <rosparam command="load" file="$(find openag_brain)/launch/personal_food_computer_v2.yaml" />
+  <rosparam command="load" file="$(find openag_brain)/launch/personal_food_computer_var_types.yaml" />
```

Comment:
A small stylistic request: let's put all of these under an `environmental_variables` namespace. This can be done with the `ns` attribute. See http://wiki.ros.org/roslaunch/XML/rosparam. Will require changing the `get_param` calls in the other files.